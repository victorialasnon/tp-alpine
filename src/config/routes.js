import {
  Version,
  Color,
  Rims,
  Upholstery,
  Equipment,
  Accessories,
  Summary,
} from '../components/Placeholders';

export default [
  {
    path: '/version',
    event: 'reset',
    name: 'Version',
    Component: Version,
  },
  { path: '/color', event: 'color', name: 'Couleur', Component: Color },
  { path: '/rims', event: 'rims', name: 'Jantes', Component: Rims },
  {
    path: '/upholstery',
    event: 'upholstery',
    name: 'Sellerie',
    Component: Upholstery,
  },
  {
    path: '/equipment',
    event: 'equipment',
    name: 'Equipements',
    Component: Equipment,
  },
  {
    path: '/accessories',
    event: 'accessories',
    name: 'Accessoires',
    Component: Accessories,
  },
  {
    path: '/summary',
    event: 'submit',
    name: 'RÃ©capitulatif',
    Component: Summary,
  },
];
