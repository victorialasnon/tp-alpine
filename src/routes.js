import {
  Version,
  Color,
  Rims,
  Upholstery,
  Equipment,
  Accessories,
  Summary,
} from './components/Placeholders';

export default [
  { path: '/version', name: 'Version', Component: Version },
  { path: '/color', name: 'Couleur', Component: Color },
  { path: '/rims', name: 'Jantes', Component: Rims },
  { path: '/upholstery', name: 'Sellerie', Component: Upholstery },
  { path: '/equipment', name: 'Equipements', Component: Equipment },
  { path: '/accessories', name: 'Accessoires', Component: Accessories },
  { path: '/summary', name: 'RÃ©capitulatif', Component: Summary },
];
