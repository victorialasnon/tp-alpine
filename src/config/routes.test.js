import routes from './routes';

test('exports correct routes', () => {
  expect(routes.map(({ name, path }) => `${name}: ${path}`)).toMatchSnapshot();
});
