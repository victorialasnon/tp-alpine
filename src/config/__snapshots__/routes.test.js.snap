// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports correct routes 1`] = `
Array [
  "Version: /version",
  "Couleur: /color",
  "Jantes: /rims",
  "Sellerie: /upholstery",
  "Equipements: /equipment",
  "Accessoires: /accessories",
  "RÃ©capitulatif: /summary",
  "reset: /summary",
  "up: /summary",
  "down: /summary",
  "confirm: /summary",
  "cancel: /summary",
]
`;
